src/cdk/v2/destinations/rakuten/data/propertiesMapping.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "sourceKeys": "properties.orderId",
```

Comment:
I remember we had some required properties, should we mark them required?

Reply:
I see land is marked. looks fine

Reply:
Already marked